### PR TITLE
sharan1404 cv bootcamp submission v1

### PR DESCRIPTION
**Pull request by sharan1404**

### CODE REVIEW

**Comment by sharan1404** [comment]
I realize it's pretty weird that the validation loss is always a lot lower than the training loss, but I honestly couldn't figure out why. Do you have any possible explanations for this?

**Comment by aydanjiwani** [comment]
code looks good overall, just one small thing- would like to see the test loss included on the end plot instead of just train and valid

**Comment by aydanjiwani** [comment]
approved, welcome to the team!
